web/docs/introduction/migrate-from-11.md

```diff
@@ -438,6 +438,322 @@ The migration functions provided below are written with the typical use cases in
 
 ##### Username & Password
 
+:::caution Users will need to migrate their password
+There is a breaking change between the old and the new auth in the way the password is hashed. This means that users will need to migrate their password after the migration, as the old password will no longer work.
+
+Since users can't reset their password by receiving an email when using the Username & Password auth method, you will need to provide a custom page to migrate the password.
+:::
+
+<details>
+  <summary>
+    Steps to create a custom page for migrating the password
+  </summary>
+
+  1. You will need to install the `secure-password` and `sodium-native` packages to use the old hashing algorithm:
+
+  ```bash
+  npm install secure-password sodium-native@3.3.0 --save-exact
```

Comment:
Maybe we should also provide version of secure-password here. I know it hasn't been updated in 4 years, but in theory it could be updated tomorrow and then we might have problems.